cli/fossilize_replay.cpp

```diff
@@ -2660,6 +2667,8 @@ struct ThreadedReplayer : StateCreatorInterface
 			       memory_index;
 		};
 
+		auto parsed_parents = make_shared<unordered_set<Hash>>();
```

Comment:
Is this really needed?

Reply:
It's needed to avoid calling `enqueue_parent_pipelines` for pipelines that were already parsed.